controller/src/main/java/io/pravega/controller/fault/ControllerClusterListener.java

```diff
@@ -81,29 +81,30 @@ protected void startUp() throws InterruptedException {
                 switch (type) {
                     case HOST_ADDED:
                         // We need to do nothing when a new controller instance joins the cluster.
-                        log.info("Received controller cluster event: {} for host: {}", type, host);
+                        log.info("Received HOST_ADDED cluster event: {} for host: {}", type, host);
                         break;
                     case HOST_REMOVED:
-                        log.info("Received controller cluster event: {} for host: {}", type, host);
+                        log.info("Received HOST_REMOVED cluster event: {} for host: {}", type, host);
                         handleHostRemoved(host);
                         break;
                     case ERROR:
                         // This event should be due to ZK connection errors. If it is session lost error then
                         // ControllerServiceMain would handle it. Otherwise it is a fleeting error that can go
                         // away with retries, and hence we ignore it.
-                        log.info("Received error event when monitoring the controller host cluster, ignoring...");
+                        log.warn("Received error event from controller host {}", host);
```

Comment:
this can be at debug infact.. it doesnt even need a user's attention.

Reply:
Though an easily tolerable one, this is a glitch during application execution and so I think it should be logged at `warn` and not `info`. Messages logged at `info` should be for events that are expected to happen in `normal` course of execution of the application.

Reply:
fixed.